db/schema.rb

```diff
@@ -1216,9 +1217,9 @@
     t.datetime "updated_at", null: false
     t.string "username"
     t.string "website_url"
-    t.string "youtube_url"
     t.boolean "welcome_notifications", default: true, null: false
     t.datetime "workshop_expiration"
+    t.string "youtube_url"
```

Comment:
I think we can keep this change in as it keeps popping up, there was a problem of ordering of fields I think

Reply:
+1 for this!!! I was just lamenting today why this keeps popping up in my diff and @rhymes pointed me here! Just wanted to say thank you for fixing this, @citizen428